yt/docker/ytsaurus/Dockerfile

```diff
@@ -116,6 +116,9 @@ RUN chmod 755 /setup_cluster_for_chyt.sh
 
 FROM base-ytsaurus-python-packages AS query-tracker-bloated
 
+# Libpython shared objects. TODO(aleksandr.gaev) use stubs instead.
```

Comment:
What does this TODO mean?

Reply:
It means that we don't need working python and `libpython3.N.so` files in this image, we only need stubs that declare required symbols

I was creating stubs using this script(passing libpython3.N.so as a first argument):
```
#/usr/bin/env bash
SYMBOLS=$(nm -D --defined-only "$1" | awk '{print $3}')
NAME=$(basename "$1")
echo "#include <stdlib.h>" > ${NAME}_stub.c
for SYMBOL in $SYMBOLS
do
        echo "void $SYMBOL(){exit(42);};" >> ${NAME}_stub.c
done

cc -shared ${NAME}_stub.c -o ${NAME}
```
and it worked on an internal cluster

Reply:
We may implement something similar to `YQL-7971`, when stubs are created automatically and added to yql_agent
In my tests I added them to `/usr/local/lib` and it made it worse for some other tools that required libpython (gdb)